Remotion/Core/Core/Utilities/DateOnlyConverter.cs

```diff
@@ -0,0 +1,73 @@
+// SPDX-FileCopyrightText: (c) RUBICON IT GmbH, www.rubicon.eu
+// SPDX-License-Identifier: LGPL-2.1-or-later
+using System;
+using System.ComponentModel;
+using System.Diagnostics.CodeAnalysis;
+using System.Globalization;
+
+namespace Remotion.Utilities;
+
+public class DateOnlyConverter : TypeConverter
+{
+  private readonly Type _dateTimeType = typeof(DateTime);
+  private readonly Type _nullableDateTimeType = typeof(DateTime?);
+  private readonly Type _nullableDateOnlyType = typeof(DateOnly?);
+  private readonly Type _dateOnlyType = typeof(DateOnly);
```

Comment:
Unrelsolved the convo since it's still in the PR as instance property. If this is intentional, please add reply. If it's gone after already done code has been pushed, just re-resolve :)